cmd/utils.go

```diff
@@ -0,0 +1,48 @@
+package cmd
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/kong/deck/utils"
+)
+
+// confirm prompts a user for a confirmation with message
+// and returns true with no error if input is "yes" or "y" (case-insensitive),
+// otherwise false.
+func confirm(message string) (bool, error) {
```

Comment:
Have you considered using a library?

For example:

https://pkg.go.dev/gopkg.in/AlecAivazis/survey.v1#Confirm

Reply:
We don't do much here so I avoided adding any dependencies.